sample/ziti-socket-sample.py

```diff
@@ -0,0 +1,40 @@
+#  Copyright (c) 2022.  NetFoundry, Inc.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#  https://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+import openziti
+from socket import SOCK_STREAM,SHUT_WR
+
+
+if __name__ == '__main__':
+    print("Ziti SDK version = {0}".format(openziti.version()))
+    sock = openziti.socket(type = SOCK_STREAM)
```

Comment:
This doesn't work for me with latest main. Should this be `openziti.zitisock.ZitiSocket(type=SOCK_STREAM)`?

Reply:
are you using python3? Python2.7 is not supported yet
When you say _doesn't work_ what do you mean?

Reply:
added `openziti.socket()` back